keyboards/crkbd/keymaps/cameronjlarsen/keymap.c

```diff
@@ -0,0 +1,363 @@
+/* Copyright 2022 Cameron Larsen <@cameronjlarsen>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include <stdio.h>
+#include "features/oneshot.h"
+
+enum layers {
+    _QWERTY = 0,
+    SYM,
+    NAV,
+    FUN,
+};
+//  Aliases for readability
+#define QWERTY DF(_QWERTY)
+#define LA_SYM LT(SYM, KC_TAB)
+#define LA_NAV MO(NAV)
+
+// One shot mods
+enum keycodes {
+    OS_SHFT = SAFE_RANGE,
```

Comment:
```suggestion
    OS_SHFT = QK_USER,
```

Reply:
I'm not familiar with this and couldn't find any references in the docs. Is this change needed because they reference the oneshot file in my user space?